sakura_core/cmd/CViewCommander_Edit_advanced.cpp

```diff
@@ -956,7 +956,7 @@ void CViewCommander::Command_MERGE(void)
 void CViewCommander::Command_Reconvert(void)
 {
 	//サイズを取得
-	int nSize = m_pCommanderView->SetReconvertStruct(NULL);
+	LRESULT nSize = m_pCommanderView->SetReconvertStruct(nullptr);
```

Comment:
対処不要) 左側、変数宣言 int を LRESULT に変更するのは悪手。

縮小変換の警告が気になるならキャスト (int) を追加するか、
SetReconvertStructの戻り値型を int に変更するのがよいはず。

コードの近代化を意識するならこうなるような。

```suggestion
	const auto nSize = m_pCommanderView->SetReconvertStruct(nullptr);
```